save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -44,10 +46,20 @@ class VulnerabilityController(
     @ApiResponse(responseCode = "200", description = "Successfully fetched all vulnerabilities by filters")
     fun getAllVulnerabilities(
         @RequestBody filters: VulnerabilityFilter,
+        authentication: Authentication?,
     ): Mono<VulnerabilityDtoList> = blockingToMono {
-        vulnerabilityService.getFiltered(filters).map(
-            Vulnerability::toDto
-        )
+        when (filters.active) {
+            true -> vulnerabilityService.getFiltered(filters).map(Vulnerability::toDto)
```

Comment:
true and else looks similar, is it expected?

Reply:
Yes. We need to check if the user has access to the data, depending on the filters configured by him.